lua/telescope/_extensions/command_palette.lua

```diff
@@ -7,7 +7,12 @@ local action_state = require "telescope.actions.state"
 local entry_display = require('telescope.pickers.entry_display')
 
 local categories
-CpMenu = CpMenu or {}
+local CpMenu = require('command_palette').CpMenu
+
+local function setup(cpMenu)
+  require('command_palette').CpMenu = cpMenu or {}
```

Comment:
`cpMenu`?

Reply:
To differentiate from `CpMenu`, it’s the function parameter. It can be renamed to whatever you like.

Reply:
No. it's good.